.github/workflows/ibmi-smoke-tests.yml

```diff
@@ -0,0 +1,55 @@
+name: Smoke Test (ibmi)
+on:
+  # schedule:
+  #   - cron: '0 9 */5 * *'
+  push
+
+jobs:
+  install-and-build:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [14]
+    name: Upload install and build artifact (Node ${{ matrix.node-version }})
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@8f4b7f84864484a7bf31766abe9204da3cbe65b3 # v3.5.0
+      - uses: actions/setup-node@64ed1c7eab4cce3362f8c340dee64e5eaeef8f7c # v3.6.0
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: yarn
+      - name: Install dependencies and build sequelize
+        run: yarn install --frozen-lockfile
+      - name: Build sequelize
+        run: yarn build
+      - name: Compress artifact
+        run: tar -cf install-build-node-${{ matrix.node-version }}.tar ./packages/core/lib ./packages/core/node_modules ./packages/core/types ./packages/validator-js/lib ./packages/validator-js/node_modules ./packages/validator-js/types ./node_modules
+      - uses: actions/upload-artifact@0b7f8abb1508181956e8e162db84b466c27e18ce # v3.1.2
+        with:
+          name: install-build-artifact-node-${{ matrix.node-version }}
+          path: install-build-node-${{ matrix.node-version }}.tar
+          retention-days: 1
+  test-ibmi:
+    strategy:
+      fail-fast: false
+      matrix:
+        node-version: [14]
+    name: ibmi (Node ${{ matrix.node-version }})
+    runs-on: ubuntu-latest
+    needs: [ install-and-build ]
+    env:
+      DIALECT: ibmi
+      SEQ_IBMI_CONN_STR: ${{ secrets.IBMI_CONN_STR }}
```

Comment:
@markdirish do you remember what `IBMI_CONN_STR` you sent to Sascha on Slack? We don't have enough history going back to that and without it getting this to work is difficult. Feel free to reach out to me on Slack